Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -87,11 +99,44 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
     return nil;
 }
 
+- (BOOL)enableVideoSending:(BOOL)enable forCall:(OCTCall *)call error:(NSError **)error
+{
+    OCTToxAVVideoBitRate bitrate = (enable) ? kDefaultVideoBitRate : kOCTToxAVVideoBitRateDisable;
+    BOOL success = [self setVideoBitrate:bitrate forCall:call error:error];
+
+    if (! success) {
+        return NO;
+    }
+
+    if (enable && (! [call isPaused])) {
+        OCTFriend *friend = [call.chat.friends firstObject];
+        self.videoEngine.friendNumber = friend.friendNumber;
+        [self.videoEngine startSendingVideo];
+    }
+    else {
+        [self.videoEngine stopSendingVideo];
```

Comment:
There is some code duplication in this method and `startEngineAndTimer:forCall:`. I'm thinking if we can refactor all this code. Can we use single method `updateAudioEngines` that will
- get active call
- check if audio is required for call
  - audio required, check if audioEngine is running and start it if needed
  - audio not required, check if audioEngine is off and stop it if needed
- check if video is required for call
  - -||-
  - -||-

In this case all over the submanager you'll need only to update active call and call `updateAudioEngines` method